cap_rds_schema.md

```diff
@@ -0,0 +1,28 @@
+# RDS file schema
+
+## AnnData fields mapping to Seurat5
+
+In the table below `seurat_obj` represents Seurat5 object, the result of the conversion. It consists of only one assay, which represents an `Assay5` object and is defined as `assay <- seurat_obs[["RNA"]]`
```

Comment:
In the table below `seurat_obj` represents Seurat5 object, --> 

In the table below `seurat_obj` represents a Seurat5 object,

Reply:
fixed